src/SprykerSdk/Acp/AcpFactory.php

```diff
@@ -241,6 +243,14 @@ protected function createManifestPagesFileValidator(): FileValidatorInterface
         return new PagesFileValidator($this->getConfig());
     }
 
+    /**
+     * @return \SprykerSdk\Acp\Validator\FileValidatorInterface
+     */
+    protected function createBusinessModesValidator(): FileValidatorInterface
```

Comment:
Just do a search and replace of BusinessModes with BusinessModel....

Reply:
seems like my keyboard is dying....